### PR TITLE
[WIP] add support for guard plugin in explorer web app

### PR DESCRIPTION
**Pull request by plouc**
- [x] Cookie token storage
- [x] Login form
- [x] Prevent access to dashboard until there's an available token
- [x] Logout

### CODE REVIEW

File: explorer/src/Api.js [hunk 1]
```diff
@@ -9,12 +12,30 @@ const JSON_HEADERS = {
 const API_BASE_URL = 'http://localhost:2405/';
```
Comment (rande) [inline explorer/src/Api.js]: the url should not contain the final `/`. `/` is also a resource.

File: explorer/src/Api.js [hunk 2]
```diff
@@ -9,12 +12,30 @@ const JSON_HEADERS = {
...
 
 const Api = {
+    login(credentials) {
+        const url = `${API_BASE_URL}login`;
```
Comment (rande) [inline explorer/src/Api.js]: `${API_BASE_URL}/login`

**Comment by plouc** [comment]
@rande, PR updated

**Comment by rande** [comment]
Can you squash your commits ? and use the syntax `feat(explorer): add support for guard plugin in explorer web app` for the commit message.

Thanks!!!

**Comment by plouc** [comment]
@rande done

**Comment by rande** [comment]
Thanks!

**Pull request merged**